docs/calico-with-docker/docker-network-plugin/ManualSetup.md

```diff
@@ -37,13 +37,13 @@ up a cluster.
 > NOTE: If you are running etcd with SSL/TLS, see the (Etcd Secure Cluster)[../../EtcdSecureCluster.md]
 > documentation.
 
-### Docker
+### Docker (with multi-host networking)
 
 Follow the instructions for installing [Docker][docker].  A version of 1.9 or
 greater is required.
 
-To use the multi-host native networking feature of Docker, the Docker daemon
-needs to be run specifying a cluster store.  If using etcd as a cluster store,
+To use Calico as a Docker network plugin, the Docker daemon
```

Comment:
I think we should also update this to mentioned daemon.json.   How about something like:

To use Calico as a Docker network plugin, the Docker daemon must be configured to run using a cluster store.  For simplicity you can use the same etcd cluster that is used by Calico, although it is not necessary to do so.  

There are two ways to configure Docker to use a cluster store:

1) Add the following additional parameter to your Docker daemon run command.

  --cluster-store=etcd://<ETCD_IP>:2379

2) Add the cluster-store details to your daemon.json file.  By default, Docker checks for the file `/etc/docker/daemon.json`.  If this file exists, add an entry for "cluster-store".  If the file does not exist, create a new one with the following contents.

    {
      "cluster-store": "etcd://<ETCD_IP>:<PORT>"
    }